src/lib/workspace/AllAnalyses.tsx

```diff
@@ -0,0 +1,338 @@
+import {
+  Button,
+  Checkbox,
+  createMuiTheme,
+  FormControlLabel,
+  Icon,
+  makeStyles,
+  Switch,
+  ThemeProvider,
+  Tooltip,
+} from '@material-ui/core';
+import { Loading, Mesa } from '@veupathdb/wdk-client/lib/Components';
+import { ContentError } from '@veupathdb/wdk-client/lib/Components/PageStatus/ContentError';
+import { useSessionBackedState } from '@veupathdb/wdk-client/lib/Hooks/SessionBackedState';
```

Comment:
What's the difference between this and the localforage thing we're using in Mocks.ts ?

Reply:
`localforage` uses a more persistent storage by default (typically [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)). `useSessionBackedState` uses a storage that gets cleared when the tab/window is closed ([sessionState](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)).